### DISCUSSION TITLE
Does corfu work with vterm in emacs?

### DISCUSSION THREAD

**Discussion by parrotcar00** [minad/corfu#518] [Q&A]
I use vterm for terminal operations from inside emacs. I couldn't get corfu to work with tab completion in vterm. Is there an example config to use corfu with vterm?

This is the vterm package that I'm talking about: https://github.com/akermu/emacs-libvterm

**Comment by minad** [comment]
I am not familiar with vterm. Corfu relies on the presence of `completion-at-point-functions` in the given major mode. If default completion works, Corfu will work.
